salt/template.py

```diff
@@ -88,8 +88,9 @@ def compile_template(template,
         if not isinstance(input_data, (dict, OrderedDict)):
             try:
                 input_data.seek(0)
-            except Exception as exp:
-                log.error('error while compiling template \'{0}\': {1}'.format(template, exp))
+            except AttributeError:
```

Comment:
I do not remember what error I was getting that caused me to add the generalized exception handling. I was probably just being lazy about researching which exceptions could be thrown. It might have been an AttributeError, I just can't remember.